front/src/common/redux/auth/auth.reducer.js

```diff
@@ -1,8 +1,10 @@
 import * as types from './types';
+import { checkUserStatus } from '../../functions';
```

Comment:
Please, rename it to `getUserType`. It is more appropriate name

Reply:
done